### DISCUSSION TITLE
MonitorPhysicalControllerStatus in winmm.dll?

### DISCUSSION THREAD

**Discussion by rccraigb** [samuelgr/Xidi#50] [Q&A] [answered]
Is it possible to receive detach/attach notifications via the interface provided by winmm.dll?

Thank you in advance!

**Comment by samuelgr** [comment, marked as answer]
Are you asking if there is a WinMM API that allows you to receive these types of notifications? As far as I know there is no way of doing that via the joystick API that WinMM provides.
